tests/testArgumentParser.py

```diff
@@ -289,13 +290,17 @@ def testAtFile(self):
     def testLogDest(self):
         """Test --logdest
         """
-        logFile = "tests_argumentParser_testLog_temp.txt"
-        self.ap.parse_args(
-            config = self.config,
-            args = [DataPath, "--logdest", logFile],
-        )
-        self.assertTrue(os.path.isfile(logFile))
-        os.remove(logFile)
+        logFile = tempfile.NamedTemporaryFile()
+        try:
+            namespace = self.ap.parse_args(
+                config = self.config,
+                args = [DataPath, "--logdest", logFile.name],
+            )
+            namespace.log.info('test logging to the specified file')
+            self.assertTrue(os.path.isfile(logFile.name))
+            self.assertGreater(os.path.getsize(logFile.name), 0)
```

Comment:
Won't make a difference because `pex_logging` doesn't seem to use the memory tester so it's not being checked at all. It only uses `unittest` in 2 out of 9 test files.